src/components/Dashboard/TodoList.vue

```diff
@@ -591,9 +627,14 @@ export default {
       }
     },
 
-    loadChangeOfDirectors (task) {
+    async loadChangeOfDirectors (task) {
       const filing = task.task.filing
       if (filing && filing.header && filing.changeOfDirectors) {
+        const bcolErr = filing.header.paymentErrorType || null
+        let bcolObj = null
+        if (bcolErr) {
+          bcolObj = await this.getErrorObj(bcolErr)
```

Comment:
Did you consider returning the error object in the filing object so we don't have to query it separately here?

Reply:
TBD in the future

Reply:
As discussed on RocketChat, although often it's better for the API to gather up and package up all the necessary data (and not require the UI to make additional network calls), in this case the additional calls are only if there is a payment error code, so it's not a performance issue worth worrying about.

No change needed here. I'm resolving this conversation.